administrator/modules/mod_latest/src/Helper/LatestHelper.php

```diff
@@ -64,6 +64,13 @@ public static function getList(Registry &$params, ArticlesModel $model)
 		{
 			$model->setState('filter.category_id', $categoryId);
 		}
+		//set workflow filter
```

Comment:
```suggestion

		//set workflow filter
```
Sorry, I've missed the missing empty line to be added for code style.

Reply:
Comments start with a blank after th // and a capital letter. Please compare other comments.

Reply:
That's right. How could I miss that? Will adjust my suggestion.